### PR TITLE
PFSA-903

### PR DESCRIPTION
**Pull request by ezengsvmx**
@ericsvmx I ran the linter, and fixed the event handlers. Can you review again? Thanks.

### CODE REVIEW

File: js/common.js [hunk 1]
```diff
@@ -6,105 +6,137 @@ import {
   View,
   Button,
   TextInput,
-  Linking
+  Linking,
```
Comment (NeerajNemaSVMX) [inline js/common.js]: @ezengsvmx  - Do we need to put the comma for the last import component?

Comment (ezengsvmx) [inline js/common.js]: I ran eslint fix, and these were added by eslint

File: js/common.js [hunk 2]
```diff
@@ -6,105 +6,137 @@ import {
...
 
 export default class SVMXExternalApp extends Component {
   constructor(props) {
     super(props);
-    this.state = {text: ''};
-    this.state = {selectedTab: 'home' };
+    this.state = {
+      text: '',
+      selectedTab: 'home',
+      receivedText: '' };
```
Comment (ericsvmx) [inline js/common.js]: This trailing `};` should be on the next line. Also make sure `receivedText: ''` ends with `,` such as `receivedText: '',`

Comment (ezengsvmx) [inline js/common.js]: Good catch, thanks. Done.

**Comment by ericsvmx** [comment]
Looks good to me

**Pull request merged**